### ISSUE TITLE
Product Spike: Multichain Deployment

### ISSUE THREAD

**Issue by fainashalts** [ethereum-optimism/ecosystem#521]
Given developer pain points regarding deployments in a multichain environment, as outlined [here](https://www.notion.so/oplabs/Multichain-Deployment-31f6a9c1263244f8bd4a7141f1e68c05?pvs=4), we need to outline the product requirements for DevX solutions. 

Definition of Done: A PRD for multichain deployment support is completed and shared with the team.

**Comment by fainashalts** [comment]
@nitaliano do you think the SuperCLI work sort of supersedes this issue? Happy to close it if so.

**Comment by pharger** [comment]
Should we kill this issue? Seems redundent of the CLI or is this something else like parallel deploy?

**Comment by fainashalts** [comment]
Yep! I think its too broad and being handled elsewhere. Closing, thanks!
